src/OpenTok/Broadcast.php

```diff
@@ -114,6 +114,22 @@ public function __get($name)
                 return null;
         }
     }
+
+	public function isHls(): bool
+	{
+		return isset($this->data['settings']['hls']);
+	}
+
+	public function isLowLatency(): bool
+	{
+		return $this->data['settings']['hls']['lowLatency'];
+	}
+
+	public function isDvr(): bool
+	{
+		return $this->data['settings']['hls']['dvr'];
+	}
```

Comment:
Why are these implemented as functions? Other properties of the Broadcast are implemented as properties (succh as `isStopped`, `streamMode`, etc.

Reply:
Also, can please add docs (like we do for `@property boolean $isStopped`).

Reply:
done